### DISCUSSION TITLE
들여쓰기가 안됩니다.

### DISCUSSION THREAD

**Discussion by guesung** [pronist/hELLO#171] [Q&A] [answered]
안녕하세요 !! 티스토리 스킨 너무너무 잘쓰고 있는 사용자입니다.
다름이 아니라, 글을 작성할 때 리스트를 이용해서 들여쓰기를 주었는데, 글을 작성하고 보면 아래 그림처럼 들여쓰기가 안됩니다.
혹시 해결책이 있을까요 ??
### 실제 작성
![image](https://user-images.githubusercontent.com/62178788/213919474-1b3b75bb-50bf-464f-9518-dbe13c758b62.png)
### 작성 후 화면
![image](https://user-images.githubusercontent.com/62178788/213919376-b8919bff-66e6-47b4-b98c-ce2999eb1999.png)

**Comment by pronist** [comment, marked as answer]
@gueit214 버그로 확인되었습니다. 감사합니다.
